frontend/projects/upgrade/src/app/features/dashboard/feature-flags/pages/feature-flag-details-page/feature-flag-details-page-content/feature-flag-exclusions-section-card/feature-flag-exclusions-table/feature-flag-exclusions-table.component.ts

```diff
@@ -0,0 +1,20 @@
+import { ChangeDetectionStrategy, Component, Input } from '@angular/core';
+import { FeatureFlagsService } from '../../../../../../../../core/feature-flags/feature-flags.service';
+import { CommonDetailsParticipantListTableComponent } from '../../../../../../../../shared-standalone-component-lib/components/common-details-participant-list-table/common-details-participant-list-table.component';
+import { CommonModule } from '@angular/common';
+import { TranslateModule } from '@ngx-translate/core';
+
+@Component({
+  selector: 'app-feature-flag-exclusions-table',
+  standalone: true,
+  templateUrl: './feature-flag-exclusions-table.component.html',
+  styleUrl: './feature-flag-exclusions-table.component.scss',
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  imports: [CommonDetailsParticipantListTableComponent, CommonModule, TranslateModule],
+})
+export class FeatureFlagExclusionsTableComponent {
+  @Input() dataSource$ = this.featureFlagService.selectFeatureFlagExclusions$;
+  isLoading$ = this.featureFlagService.isLoadingFeatureFlags$;
```

Comment:
yes, I can create that. I got lazy and decided it would be fine to reuse one, but that would be best. We're gonna have a bunch of 'isLoading' ui flags in the store but that is more explicit and consistent.